{"name": "Невозможно авторизовать пользователя, если хотя бы одно из полей неверно", "status": "passed", "parameters": [{"name": "field", "value": "'email'"}], "start": 1719127734421, "stop": 1719127735932, "uuid": "e6bba957-6d52-4476-9f3e-4de5ab302fee", "historyId": "b8eb9af926b8187d9f932fbbb6fba126", "testCaseId": "75e5e0134f47bb9cc437e9df45383da3", "fullName": "tests.test_user.TestUserLogin#test_user_login_with_wrong_data_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "thinkpad"}, {"name": "thread", "value": "9864-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}